students/charlie_england/lesson05/assignment/src/database.py

```diff
@@ -0,0 +1,99 @@
+import csv
+import os.path
+from pymongo import MongoClient
+
+mongo = MongoClient("localhost:27017")
+
+db = mongo["hp_norton"]
+db.drop_collection("product")
+db.drop_collection("customers")
+db.drop_collection("rental")
+
+def import_data(data_dir, *files):
+    """
+        given a directory and files
+    """
+    added = [0, 0, 0]
+    errors = [0, 0, 0]
+    reference = 0
+    for filepath in files:
+        collection_name = filepath.split(".")[0]
+        with open(os.path.join(data_dir, filepath)) as file:
```

Comment:
Whenever you use os.path, be careful! Generally, you should be aware where this file is run from. If you can make sure it's always run from the same path, you will find it easier to specify the data_dir.